css-nav-1/Overview.bs

```diff
@@ -1131,105 +1155,114 @@ run the following steps:
             and is perpendicular to <var>dir</var>.
         2. For each <var>candidate</var> in <var>candidates</var>,
-            1. If the <var>searchOrigin</var> is an element, then find the points <var>P1</var> inside the <a>boundary box</a> 
-                of <var>searchOrigin</var> and P2 inside the boundary box of <var>candidate</var>.
-            2. Else if the <var>searchOrigin</var> is a point, then let the point <var>P1</var> be the <var>searchOrigin</var> 
-                and find the point <var>P2</var> inside the <a>boundary box</a>.
-        3. Let <var>intersectRect</var> be an <dfn>intersecting area</dfn> between the search origin and a candidate.
-        4. Select the <var>P1</var> and <var>P2</var> that make the smallest <var>distance</var> between those two points by 
-            <a>calculating the distance</a>.        
-        5. Return the item of the <var>candidates</var> set that has the smallest <var>distance</var>.
+            [=find the shortest distance=] between <var>searchOrigin</var> and <var>candidate</var>
+            in direction <var>dir</var>.
+        3. Return the item of the <var>candidates</var> set that has the smallest distance.
             If several have the same distance,
             return the first one in document order,
             unless its [=boundary box=] overlaps with the [=boundary box=] of an other item at the same distance,
             and that item is higher in the CSS [=painting order=].
             In that case, return that item instead,
-            unless it too is overlapped with another higher item at the same distance, recursively.    
+            unless it too is overlapped with another higher item at the same distance, recursively.
+
 </div>
 
-<div algorithm="to calculate the distance">
-To <dfn lt="calculating the distance | to calculate the distance">calculate the <var>distance</var></dfn>, the function is defined as below:
+<div algorithm="to find the shortest the distance">
+To <dfn lt="find the shortest distance">to find the shortest distance</dfn>
+between a <var>reference</var> and a <var>candidate</var>
+in direction <var>dir</var>,
+find the points <var>P1</var> and <var>P2</var>,
+respectively within the boundary boxes of the <var>reference</var> and of the <var>candidate</var>,
+that minimize the <var>distance</var> as defined as below:
 
-<pre class="prod">
+<pre><code>
     <var>distance</var> = <var>euclidean</var> + <var>displacement</var> - <var>alignment</var> - <var>sqrt(Overlap)</var>
-</pre>
+</code></pre>
 
-The meaning of each factor in the function is as follows:
+The meaning of each term is as follows:
 
 <dl dfn-type="value" dfn-for="spatial-navigation-distance-function">
     <dt><var>euclidean</var>
-    <dd>The euclidean distance between <var>P1</var> and <var>P2</var>
+    <dd>
+        The euclidean distance between <var>P1</var> and <var>P2</var>
 
     <dt><var>displacement</var>
```

Comment:
I wasn't sure that this "displacement" term is appropriate.
Does it make sense?

Reply:
Not sure either, but it is clearly defined and used, and is not exposed through any API. If a native speaker finds a better word later, we can always change it.

Reply:
:+1: